audit/forgerock-audit-handler-jms/src/test/java/org/forgerock/audit/handlers/jms/JndiJmsContextManagerTest.java

```diff
@@ -80,8 +84,7 @@ public void testContextLoading() throws Exception {
         // Setup known mocked classloader and context builder to be used for validation.
         InitialContextFactoryBuilder builder = mock(InitialContextFactoryBuilder.class);
         setInitialContextFactoryBuilder(builder);
-        ClassLoader contextClassLoader = mock(ClassLoader.class);
-        Thread.currentThread().setContextClassLoader(contextClassLoader);
+        ClassLoader contextClassLoader = Thread.currentThread().getContextClassLoader();
```

Comment:
Any chance you can put the classloader changes in a separate commit, instead of grouping the changes in with "Fix any() not matching null since Mockito 2.1.0"? I know it's extra work to redo that commit, but it can help us later when doing `git log` / `git blame`.